src/test/java/StepDefinitions/MyStepdefs.java

```diff
@@ -0,0 +1,61 @@
+package StepDefinitions;
+
+import Pages.HomePage;
+import Pages.LoginPage;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import java.time.Duration;
+
+public class MyStepdefs {
+    WebDriver driver;
+    WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+ @Before
+ public void iOpenBrowser(){
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/222356100-5dcb70b0-a00c-43f2-aa28-fb938fa683e6.png)

Hint

```
  WebDriver driver;
  WebDriverWait wait;


  @Before
  public void openBrowser() {
    WebDriverManager.chromedriver().setup();
    driver = new ChromeDriver();
    wait = new WebDriverWait(driver, Duration.ofSeconds(10));
  }
```